src/audio/volume.c

```diff
@@ -191,13 +191,14 @@ static struct comp_dev *volume_new(struct sof_ipc_comp *comp)
 
 	comp_set_drvdata(dev, cd);
 	work_init(&cd->volwork, vol_work, dev, WORK_ASYNC);
-
+	/* set volume min/max levels */
+	vol_set_min_max_levels(cd, ipc_vol->min_value, ipc_vol->max_value);
 	/* set the default volumes */
 	for (i = 0; i < PLATFORM_MAX_CHANNELS; i++) {
-		cd->volume[i] = VOL_ZERO_DB;
-		cd->tvolume[i] = VOL_ZERO_DB;
+		cd->volume[i] = cd->max_volume;
+		cd->tvolume[i] = cd->max_volume;
 	}
-
+	
```

Comment:
trailing white space, checkpatch should complain here btw.

Reply:
and it did!

Reply:
Yes, it did complain about them. I must have something wrong with formatting on my editor since it treated every new line this way. Should I remove all these trailing white spaces?

Reply:
yes, just remove training whitespaces from your patch. i.e. fix everything check patch complains about.